src/mpid/ch4/src/ch4_impl.h

```diff
@@ -551,6 +579,7 @@ do {                                                                  \
 #define MPIDI_CH4U_EPOCH_FENCE_CHECK(win, mpi_errno, stmt)              if (0) goto fn_fail;
 #define MPIDI_CH4U_EPOCH_POST_CHECK(win, mpi_errno, stmt)               if (0) goto fn_fail;
 #define MPIDI_CH4U_EPOCH_LOCK_CHECK(win, mpi_errno, stmt)               if (0) goto fn_fail;
+#define MPIDI_CH4U_EPOCH_PER_TARGET_LOCK_CHECK(win, rank, mpi_errno, stmt)  if (0) goto fn_fail;
```

Comment:
These two should exist in ch4/src/ch4_impl.h.

Actually I am going to rename some of the macros as well. ORIGIN -> ACCESS, TARGET->EXP.